Mandater/ClientApp/reducers/ElectionReducer.ts

```diff
@@ -0,0 +1,79 @@
+﻿import { Action, Reducer } from "redux";
+import { ElectionState, unloadedState } from "../interfaces/states/ElectionState";
+import { GetMenuDataAction, InitializeParliamentaryElectionAction, UpdateCalculationAction, UpdateSettingsMenuAction, ToggleAutoComputeAction } from "../actions/ElectionActions";
+import { ElectionActionEnum as ElectionActionEnum } from "../enums/ActionTypeEnums";
+
+// TODO: Make actions for updates of elections etc...
+
+type KnownAction = GetMenuDataAction
+                   | InitializeParliamentaryElectionAction
+                   | UpdateCalculationAction
+                   | UpdateSettingsMenuAction
+                   | ToggleAutoComputeAction;
+
+// NB: BaseReducer Typescript (Reducer<State>) definition changes as of redux 4.0.0
+// https://github.com/rt2zz/redux-persist/pull/778
+
+
+// ReSharper disable TsResolvedFromInaccessibleModule
+export const reducer: Reducer<ElectionState> = (state: ElectionState, incomingAction: Action) => {
```

Comment:
I suggest changing this to the same style as in TableReducer, for several reasons -- one being consistency, the other being prettiness, and the third being that functions that are frequently called should preferably not be lambdas unless it adds to readability (which it does not here).

export default function(args){} is overall more clean in my opinion, since you can then assign the reducers to state in store.